buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -203,7 +203,7 @@
       },
       "inputs": {
         "filename": "build-packages.cmd",
-        "arguments": "-BuildArch=$(Architecture) -BuildType=$(PB_BuildType) $(portableBuild)",
+        "arguments": "-BuildArch=$(Architecture) -BuildType=$(PB_BuildType) $(portableBuild) -ExtraParameters=/p:PackageCompatJit=1",
```

Comment:
`-ExtraParameters` is documented in the help message; AFAICT the ability to use `--` is not.